src/api/app/controllers/webui/packages/files_controller.rb

```diff
@@ -64,15 +65,21 @@ def update
         end
 
         if errors.blank?
-          flash.now[:success] = "'#{params[:filename]}' has been successfully saved."
+          flash.now[:success] = "'#{@filename}' has been successfully saved."
         else
-          flash.now[:error] = "Error while adding '#{params[:filename]}': #{errors.compact.join("\n")}."
+          flash.now[:error] = "Error while adding '#{@filename}': #{errors.compact.join("\n")}."
           status = 400
         end
 
         status ||= 200
         render layout: false, status: status, partial: 'layouts/webui/flash', object: flash
       end
+
+      private
+
+      def set_filename
+        @filename = params[:filename]
+      end
```

Comment:
We don't gain anything by doing this... And we're increasing amount of code... I would just use the param as it was before...

Reply:
Oh I see you wanna use that instance variable below... NVM